lib/cell-manager.coffee

```diff
@@ -15,15 +15,13 @@ module.exports = CellManager =
         {row, column} = editor.getLastCursor().getBufferPosition()
 
         if row > 0
-            range = [[0, 0], [row - 1, 100]]
+            range = [start, [row - 1, 9999999]]
```

Comment:
On 26/07/16 10:38, Lukas Geiger wrote:
> In lib/cell-manager.coffee <https://github.com/nteract/hydrogen/pull/381#discussion_r72217505>:
> 
>> @@ -15,15 +15,13 @@ module.exports = CellManager =
>>          {row, column} = editor.getLastCursor().getBufferPosition()
>>  
>>          if row > 0
>> -            range = [[0, 0], [row - 1, 100]]
>> +            range = [start, [row - 1, 9999999]]
> 
> That could lead to problems if the cursor is inside the comment defining the code cell.
> But it would be more accurate otherwise.

is it possible to detect whether we're inside a comment? I mean using the grammar.